src/components/HelmReleases/HelmReleaseDescription.js

```diff
@@ -0,0 +1,12 @@
+import { Trans } from 'react-i18next';
+import { Link as ExternalLink } from '../../shared/components/Link/Link';
+import React from 'react';
+
+export const description = (
+  <Trans i18nKey={'helm-releases.description'}>
+    <ExternalLink
+      className="bsl-link"
+      url="https://helm.sh/docs/glossary/#release"
+    />
+  </Trans>
+);
```

Comment:
All of this should be moved to one generic component which you will import and pass just `url` and `i18nKey` prop.

Reply:
done